lib/java_buildpack/framework/app_dynamics_agent.rb

```diff
@@ -193,6 +194,24 @@ def override_default_config_if_applicable
           end
         end
       end
+
+      # Check for configuration files locally. If found, copy to conf dir under each ver* dir
+      # @return [Void]
+      def override_default_config_local
+        return unless @application.environment['APPD_CONF_DIR']
+
+        app_conf_dir = @application.environment['APPD_CONF_DIR']
```

Comment:
I think we should check that `app_conf_dir` exists. If a user provided this environment variable I would assume that the directory should exist and a failure might be more helpful than a silently ignoring the config if, for example, they typoed the dir name.